tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -998,54 +858,6 @@ def test_transformations(self, connect_protocol, metadata_quorum, use_new_coordi
             assert obj['payload']['content'] in self.FIRST_INPUT_LIST
             assert obj['payload'][ts_fieldname] == ts
 
-    @cluster(num_nodes=5)
-    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=False, exactly_once_source=True, connect_protocol='sessioned')
```

Comment:
This is a weird mismash of ancient Kafka versions. But shouldn't we at least keep the test cases where we're testing DEV_BRANCH (and possibly add some non-ancient versions to test, perhaps even supported versions?)